api-samples/sandbox/sandbox/sandbox.html

```diff
@@ -0,0 +1,83 @@
+<!--
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+  -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="handlebars-1.0.0.beta.6.js"></script>
+  </head>
+  <body>
+    <script id="sample-template-1" type="text/x-handlebars-template">
+      <div class='entry'>
+        <h1>Hello</h1>
+        <p>This is a Handlebar template compiled inside a hidden sandboxed
+          iframe.</p>
+        <p>The counter parameter from postMessage (outer frame) is:
+          {{counter}}</p>
+      </div>
+    </script>
+
+    <script id="sample-template-2" type="text/x-handlebars-template">
+      <div class='entry'>
+        <h1>Welcome back</h1>
+        <p>This is another Handlebar template compiled inside a hidden sandboxed
+          iframe.</p>
+        <p>The counter parameter from postMessage (outer frame) is:
+          {{counter}}</p>
+      </div>
+    </script>
+
+    <script>
+      const templatesElements = document.querySelectorAll(
+          "script[type='text/x-handlebars-template']"
+        ),
+        templates = {},
+        source,
+        name;
```

Comment:
@sebastianbenz, it took me a minute to realize why this was valid JS. I don't think it's very typical. What do you think about only using common programming styles for future extension work? A good rule of thumb for informational materials is to avoid requiring readers to puzzle over more than one thing at a time.

@IanStanion-google, there's no request for you on this item.

Reply:
+1 we should avoid this in the future